securing_openshift_gitops/masking-sensitive-annotations-in-the-argo-cd-web-ui.adoc

```diff
@@ -0,0 +1,28 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="masking-sensitive-annotations-in-the-argo-cd-web-ui"]
+= Masking sensitive annotations in the Argo CD Web UI
+:context: masking-sensitive-annotations-in-the-argo-cd-web-ui
+
+toc::[]
+
+Argo CD hides sensitive annotation values on Secret resources from the Argo CD Web UI and CLI. Users can configure this by specifying annotation keys to be masked in the Argo CD Custom Resource (CR). This feature enhances security by preventing accidental exposure of sensitive information, such as tokens or API keys, stored in annotations on Secret resources.
```

Comment:
```suggestion
Argo CD hides sensitive annotation values on Secret resources from the Argo CD Web UI and CLI. Users can configure this by specifying annotation keys to be masked in the Argo CD custom resource (CR). This feature enhances security by preventing accidental exposure of sensitive information, such as tokens or API keys, stored in annotations on `Secret` resources.
```

IMO, "Secret" should be in back ticks. Alternatively you can use lower case.

Reply:
Fixed.